src/utils/css-supports.ts

```diff
@@ -0,0 +1,9 @@
+const cssSupports = (property: string, value: string): boolean => {
```

Comment:
It's quite verbose. Personally I'd write it as:

```js
const cssSupports = (property: string, value: string): boolean => (
  'CSS' in window && 'supports' in window.CSS ? window.CSS.supports(property, value) : true
)
```
depending on your transpiler config maybe even:
```js
const cssSupports = (property: string, value: string): boolean => (
  window.CSS?.supports?.(property, value) ?? true
)
```

but LGTM!

Reply:
Thanks for the suggestions.

The first one is not really in line with the style we use. A bit of verbosity for the sake of clarity is preferred.

For the second one, I definitely considered it, but since everything I could find about checking for browser features suggests using the `in` keyword, I'd rather do it like that. Though you're right, with our current transpiler options, pretty sure it would work.

Thanks again, there's a chance someday we'll look at it again and find it indeed too verbose 😅😅